src/test/java/common/ValueChoosers.java

```diff
@@ -24,4 +25,36 @@ public static String getRandomDate() {
 
         return listOfRandomDate.get(RandomUtils.nextInt(0, listOfRandomDate.size()-1));
     }
+
+    public static int getRandomId() {
```

Comment:
Kakvi su ovo randomId-evi :P To treba da izvlacis iz liste, preko poziva, ne ovako. Ili ovo sluzi za nesto drugo?

Reply:
Aha, video sam gde koristis, da  pravis, to im cudno malo ali okej je onda